### DISCUSSION TITLE
Integrate `yarn content move` into PR workflows for title-changing PRs

### DISCUSSION THREAD

**Discussion by yarusome** [mdn/mdn-community#433] [Platform]
Currently, moving contents around requires using `yarn content move <from-slug> <to-slug> [locale]`, and there are mainly two types of PRs need to do this:

1. Changing the paths of pages except for anything after the last slashes;
2. Editing the titles of pages.

However, it's unnecessary to require contributors to run the command manually for type 2 PRs since all the command parameters, i.e. `from-slug`, `<to-slug>`, and `locale`, can be easily deduced from the front matter and the file paths. Thus it's more ergonomic to incorporate this command into workflows for PRs that edit titles but *don't really*<sup>[1]</sup> move pages around.

A side gain is that contributors can even edit titles through the GitHub UI without having to install all the toolchains.

---

[1] The [`pr-test`](https://github.com/mdn/content/blob/main/.github/workflows/pr-test.yml#L40-L45) workflow already can tell which files are renamed.

**Comment by Josh-Cena** [comment]
Editing page titles doesn't necessarily change the slug, and I would prefer it to not by default; fewer redirects is better for maintenance.
